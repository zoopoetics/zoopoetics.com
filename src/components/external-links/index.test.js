import React from 'react';
import renderer from 'react-test-renderer';
import ExternalLinks from './index';

jest.mock('styled-icons/fa-brands/Github', () => ({
  Github: () => 'Github',
}));
jest.mock('styled-icons/fa-brands/Linkedin', () => ({
  Linkedin: () => 'Linkedin',
}));
jest.mock('styled-icons/fa-brands/Twitter', () => ({
  Twitter: () => 'Twitter',
}));

describe('ExternalLinks', () => {
  let component;
  let instance;

  beforeEach(() => {
    component = renderer.create(<ExternalLinks />);
    instance = component.getInstance();
  });

  it('matches snapshot', () => {
    expect(component.toJSON()).toMatchSnapshot();
  });
});
