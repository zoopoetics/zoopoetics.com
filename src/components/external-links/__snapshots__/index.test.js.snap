// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExternalLinks matches snapshot 1`] = `
.c0 {
  box-sizing: border-box;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  list-style: none;
  padding: 0;
}

.c2 {
  color: white;
}

.c1 {
  margin: 1rem;
}

<nav
  className=""
>
  <ul
    className="c0"
  >
    <li
      className="c1"
    >
      <a
        className="c2"
        href="https://www.linkedin.com/in/zoopoetics/"
        target="_blank"
      >
        Linkedin
      </a>
    </li>
    <li
      className="c1"
    >
      <a
        className="c2"
        href="https://github.com/zoopoetics/"
        target="_blank"
      >
        Github
      </a>
    </li>
  </ul>
</nav>
`;
